enhancements/ingress/configurable-dns-loadbalancerservice.md

```diff
@@ -136,13 +137,14 @@ of type `LoadBalancerDNSManagementPolicy`. This field will default to `Managed`.
 type LoadBalancerStrategy struct {
     // <snip>
 
-    // dnsManagementPolicy indicates if the lifecyle of the wildcard DNS record
+    // dnsManagementPolicy indicates if the lifecycle of the wildcard DNS record
     // associated with the load balancer service will be managed by
     // the ingress operator. It defaults to Managed.
-    // Valid values are: Managed and Unmanaged. 
+    // Valid values are: Managed and Unmanaged.
     //
     // +kubebuilder:default:="Managed"
-    // +kubebuilder:validation:Optional
+    // +kubebuilder:validation:Required
+    // +default="Managed"
```

Comment:
What's the motivation behind this optional to required change? Does it actually change DNS managed functionality (since it always defaulted to Managed anyways)? Or it just for API verbosity?

Reply:
Ah nvm, I just realized this is a house-keeping update, as you are just matching what the API currently is.